app/resources/item.py

```diff
@@ -0,0 +1,96 @@
+import json
+import random
+from flask_restful import Resource, abort, marshal_with, reqparse, request, fields
+
+from common.db import execute_query, execute_sql_query, select_table, check_exists
+
+item_fields = {
+    'ID': fields.String,
+    'Status': fields.String,
+    'ItemType': fields.String,
+    'ItemName': fields.String,
+    'EditionNumber': fields.String,
+    'Ages': fields.String,
+    'NumberOfParts': fields.String,
+    'Location': fields.String,
+    'OrderingCompany': fields.String,
+    'TestID': fields.String,
+}
+
+class Item(Resource):
+
+    @marshal_with(item_fields)
+    def get(self, id):
+        """
+        Retrieve item with id
+        ---
+        tags:
+          - Items
+        parameters:
+          - in: path
+            name: id
+            type: string
+            required: true
+        responses:
+          200:
+            description: Item with given id
+            schema:
+              type: array
+              items:
+                $ref: '#/definitions/Item'
+          500:
+            description: Error fetching items
+        definitions:
+          Item:
```

Comment:
Is the `Item` definition complete ? It doesn't seem to fully describe the `item_fields` definition